tests/test_integration.py

```diff
@@ -957,6 +996,7 @@ def test_pex_multi_resolve_2():
 
 @contextmanager
 def pex_manylinux_and_tag_selection_context():
+    # type: () -> Iterator[Tuple[Callable[[str, str, str, str, Optional[str]], None], Callable[[str, str, str, str], None]]]
```

Comment:
Absolutely disgusting. But we can't use type aliases because those require setting variables at runtime, and that would break Python 2.

Reply:
This is untrue. You can define the inside the `if TYPE_CHECKING` guard, and, if you need to cast to them, pass those type aliases to cast as strings.